Added zkEVM support details (#1145)
maticnetwork/matic-docs: docs/zkEVM/develop.md

  <source type="video/mp4" src="/img/zkevm/tutorial.mp4"></source>
  <p>Your browser does not support the video element.</p>
</video>

## **zkEVM Support**

> If you need help with anything related to the Polygon zkEVM, you can raise a ticket on [Polygon Support](https://support.polygon.technology/support/tickets/new) portal. Additionally, you can reach out to the support team available on **#zkevm-support** channel on our Discord server. Instructions for raising zkEVM Testnet support ticket are as follows:

Reviewer: ```suggestion
If you need help with anything related to the Polygon zkEVM, you can raise a ticket on the [Polygon Support](https://support.polygon.technology/support/tickets/new) portal. Additionally, you can reach out to the support team available on the **#zkevm-support** channel on the [Polygon Discord server](https://discord.com/invite/XvpHAxZ). Instructions for raising a zkEVM testnet support ticket are as follows:
```

> 1. Join the **Polygon Discord** server [here](https://discord.gg/0xPolygon)

Reviewer: ```suggestion
1. Join the **Polygon Discord** server [here](https://discord.gg/0xPolygon).
```

> 2. Accept the invite sent via DM

Reviewer: ```suggestion
2. Accept the invite sent via DM.
```

> 3. Take **Member** role under **#roles**

Reviewer: ```suggestion
3. Take the **Member** role under **#roles**.
```

> 4. Go to the **#zkevm-support** channel

Reviewer: ```suggestion
4. Navigate to the **#zkevm-support** channel.
```
